test/unit/lib/aws/request.test.js

```diff
@@ -246,7 +271,10 @@ describe('#request', () => {
       const awsRequest = proxyquire('../../../../lib/aws/request', {
         'aws-sdk': { S3: FakeS3 },
       });
-      return expect(awsRequest({ name: 'S3' }, 'error')).to.be.rejected;
+      return expect(awsRequest({ name: 'S3' }, 'test')).to.eventually.be.rejected.and.have.property(
```

Comment:
is return needed here? should `it` be marked as async and awaited?

Reply:
Technically it's needed if we do not `await` (and effect is same), but I agree that it's better to mark all promise returning functions as _async_, and to use `await` when we do not intend to return anything substantial.

I've updated those tests